tests/unit/sample_ip_ranges.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
This is a subset of what actually gets pulled from https://ip-ranges.amazonaws.com/ip-ranges.json
I filtered it down so that there's only 1 entry per region